src/plugins/data/public/plugin.ts

```diff
@@ -163,9 +164,10 @@ export class DataPublicPlugin
       }))
     );
 
-    const ac = this.autocomplete.setup(core);
-    ac.addQuerySuggestionProvider('SQL', getSQLSuggestions);
-    ac.addQuerySuggestionProvider('kuery', getDQLSuggestions);
+    const autoComplete = this.autocomplete.setup(core);
+    autoComplete.addQuerySuggestionProvider('SQL', getSQLSuggestions);
+    autoComplete.addQuerySuggestionProvider('kuery', getDQLSuggestions);
+    autoComplete.addQuerySuggestionProvider('PPL', getPPLSuggestions);
```

Comment:
nit: we should move adding the suggestion provider for PPL and SQL to the query enhancements plugin since registering the language happens there as well. Making the suggestion provider also be a part of the language service's register function would bring all these language related features into one place too.